Tools/AP_Periph/Parameters.h

```diff
@@ -95,7 +95,7 @@ class Parameters {
 
 #ifdef HAL_PERIPH_ENABLE_GPS
     AP_Int8 gps_port;
-#if HAL_NUM_CAN_IFACES >= 2
+#if HAL_NUM_CAN_IFACES >= 2 || GPS_MOVING_BASELINE
```

Comment:
I think just having GPS_MOVING_BASELINE is enough.

Reply:
done